docs/components/files/test-drop-fences.html

```diff
@@ -0,0 +1,51 @@
+---
+title: Testing - Drop Fences
+---
+{% extends 'test.njk' %}
+{% block content %}
+  <section>
+    <nav class="hxBreadcrumb">
+      <a href="components/files">Files</a>
+      <hx-icon class="delimiter" type="angle-right"></hx-icon>
+      <a href="#">{{page.title}}</a>
+    </nav>
+  </section>
+
+  <section>
+    <h2>Drop Fence (Empty)</h2>
+    <hx-drop-fence>
+    </hx-drop-fence>
+  </section>
+
+  <section>
```

Comment:
We'll probably want some test cases for a fence with text node children.

```html
<hx-drop-fence>I'm plain text</hx-drop-fence>
```

Reply:
added -- look below in file